db/seeds.rb

```diff
@@ -1,7 +1,323 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+Editor.create(
+  kind: "board",
+  title: "Editor-in-Chief",
+  first_name: "Arfon",
+  last_name: "Smith",
+  login: "arfon",
+  email: "",
+  avatar_url: "http://joss.theoj.org/assets/editors/arfon-33e4ac87ed2e073216fda0d3383b25fc3a15cc7ac5e70aab3e9f85f68b3d4a65.jpg",
+  categories: [],
+  url: "http://arfon.org/",
+  description: <<-STR.strip_heredoc()
+    A lapsed academic with a passion for new models of scientific collaboration, he's used big telescopes
+    to study <a href="http://www.arfon.org/thesis" target="_blank">dust in space</a>, built
+    sequencing pipelines in <a href="http://www.sanger.ac.uk/" target="_blank">Cambridge</a> and
+    engaged millions of people in online citizen science by co-founding the
+    <a href="http://www.zooniverse.org" target="_blank">Zooniverse</a>.
+  STR
+)
+
+Editor.create(
```

Comment:
Shouldn't these all be `kind: topic`?

Reply:
Yeah, it's the column default, but there's no harm in being explicit here.